canonical/juju-dashboard
# WD-3854 - split table utils

## Done

- Make some shared table functions to be components.
- Split utils for extracting data from a model.

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Go to the model list.
- Check that data in the tables is unchanged.

## Details

https://warthogs.atlassian.net/browse/WD-3854
